pkg/sdk/tasks_impl_gen.go

```diff
@@ -226,6 +228,9 @@ func (r *CreateOrAlterTaskRequest) toOpts() *CreateOrAlterTaskOptions {
 			UserTaskManagedInitialWarehouseSize: r.Warehouse.UserTaskManagedInitialWarehouseSize,
 		}
 	}
+	if r.Config != nil {
+		opts.Config = String(fmt.Sprintf("$$%s$$", *r.Config))
```

Comment:
should it be added to the migration notesa too? (I mean if someone had dollar signs added by hand previously?)

Reply:
True, added